client/src/test/java/com/sinch/sdk/SinchClientTest.java

```diff
@@ -57,4 +59,48 @@ void defaultSmsUrlAvailable() {
     SinchClient client = new SinchClient(configuration);
     assertNotNull(client.getConfiguration().getSmsUrl());
   }
+
+  @Test
+  void defaultVerificationUrlAvailable() {
+    Configuration configuration =
+        Configuration.builder().setKeyId("foo").setKeySecret("foo").setProjectId("foo").build();
+    SinchClient client = new SinchClient(configuration);
+    assertNotNull(client.getConfiguration().getVerificationUrl());
```

Comment:
Instead of checking for not null, you can check for the real value. For the verification API, it's fine as there is only one URL, but for voice, you may want to check it uses the default one

Reply:
The test is against checking a default value is present
The defaultVoiceUrl test is checking against a not empty value
voiceUrlFromRegion test is the only we are checking against a value coming from configuration file. Ideally we should set for test such value to not depend of configuration values
So, back to the current test, this is why it is not checking against a hard coded value... it is not good enough

Reply:
Ok, it's true you are using a configuration file. On my side, I've chosen to hardcode the URL from the specification and to provide a default value, but I understand your point :)

Reply:
Let's add memes in our PRs :)

![image](https://github.com/sinch/sinch-sdk-java/assets/142824551/a723ceb1-0d0c-4ed5-8f57-b8c4e9ed2e66)